log/src/main/java/kcl/teamIndexZero/traffic/log/FileIO/FileInput.java

```diff
@@ -73,11 +73,31 @@ public FileInput(String folder_path, String file_name) throws InvalidPathExcepti
     }
 
     /**
+     * Closes the reader
+     *
+     * @return Success
+     */
+    public synchronized boolean closeReader() {
+        if (this.reader != null) {
+            try {
+                reader.close();
+                this.reader = null;
+                return true;
+            } catch (IOException e) {
+                MicroLogger.INSTANCE.log_Error("IOException raised in [FileInput.closeReader()] for , ", super.getFilePath());
+                MicroLogger.INSTANCE.log_ExceptionMsg(e);
+                return false;
```

Comment:
Cool, I'll see about modifying it if I get a little bit of time before wed. Thx.